packages/signed-token/src/index.ts

```diff
@@ -0,0 +1,120 @@
+import _ from 'lodash';
+import hmacSha256 from 'crypto-js/hmac-sha256';
```

Comment:
Is there a reason we use this and not `createHmac` from the native `node:crypto` module?

Reply:
Great question! I can only assume that I missed the fact that node includes HMAC support since v0.1.94 🤦

Can we convert to using node's hmac now? Ideally we'd want them to be bitwise compatible, but they might be if we specify sha256 as the algorithm?

Reply:
they should be, I can write a quick script to verify that they match for a couple inputs.

Reply:
I think I've convinced myself that they should be the same:

```ts
import hmacSha256 from 'crypto-js/hmac-sha256.js';
import crypto from 'node:crypto';

const KEY = '12345abcde';
const values = ['', 'foo', 'bar', '{"foo":"bar"}'];

for (const value of values) {
  const oldAlg = hmacSha256(value, KEY).toString();
  const newAlg = crypto.createHmac('sha256', KEY).update(value).digest('hex');
  if (oldAlg !== newAlg) {
    console.log(`Error: ${oldAlg} !== ${newAlg}`);
  }
}
```

Reply:
Nice! It's also a good test of the algorithm implementations 😄